README.md

```diff
@@ -107,7 +125,7 @@ run.yml
   tasks:
     - name: Network Backup and Resource Manager
       ansible.builtin.include_role:
-        name: network.backup.run
+        name: network.backup.backup
       vars:
         operation: backup
```

Comment:
Do we still need the `backup` key even though we're invoking the `backup` role directly?

Reply:
Yeah so we still want to have the `operation` key, even though this would end up as default and need not to be provided with the playbook(would be in separate PR), This includes maintaining the operation key to ensure future scalability and consistency with the Network Validated Content pattern.
For example, in future, we still want to add operations like `verify` and `schedule`.